src/pages/en/guides/data-fetching.md

```diff
@@ -44,11 +118,11 @@ const data = fetch('https://example.com/movies.json').then((response) =>
 );
```

Comment:
I know this isn’t one of your additions, but does the fetch call here not need an `await`?

Nominating @Princesseuh as a source of truth, if you have a moment.

Reply:
@RafidMuhymin popped in to the docs channel and suggested await would be consistent with the other examples, and I trust whatever he says! So... what if I just sneak it in there and see if anyone complains?  :)